cmd/tink-cli/cmd/get/get.go

```diff
@@ -0,0 +1,100 @@
+package get
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/jedib0t/go-pretty/table"
+	"github.com/spf13/cobra"
+)
+
+type CmdOpt struct {
```

Comment:
```suggestion
type Options struct {
```
so it looks like `get.Options` and reads better imo.